docs/OMF.rst

```diff
@@ -179,28 +77,174 @@ Connect the new application to the PI Connector Relay by selecting the new Fledg
-   - **Producer Token:** The Producer Token provided by the PI Relay Data Connection Manager.
-   - **Data Source:** Defines which data is sent to the PI Connector Relay. Choices are: readings or statistics (that is, Fledge's internal statistics).
-   - **Static Data:** Data to include in every reading sent to PI.  For example, you can use this to specify the location of the devices being monitored by the Fledge server.
-- Connection management (These should only be changed with guidance from support)
-   - **Sleep Time Retry:** Number of seconds to wait before retrying the HTTP connection (Fledge doubles this time after each failed attempt).
-   - **Maximum Retry:** Maximum number of times to retry connecting to the PI server.
-   - **HTTP Timeout:** Number of seconds to wait before Fledge will time out an HTTP connection attempt.
-- Other (Rarely changed)
-   - **Integer Format:** Used to match Fledge data types to the data type configured in PI. This defaults to int64 but may be set to any OMF data type compatible with integer data, e.g. int32.
-   - **Number Format:** Used to match Fledge data types to the data type configured in PI. The default is float64 but may be set to any OMF datatype that supports floating point values.
-   - **Compression:** Compress the readings data before sending it to the PI System.
+Now go to the Fledge user interface, create a new North instance and select the “OMF” plugin on the first screen. Continue with the configuration, choosing the connector relay as the end point to be connected.
+
+OSISoft Cloud Services
+~~~~~~~~~~~~~~~~~~~~~~
+
+The original cloud services from OSISoft, this has now been superseded by AVEVA Data Hub, and should only be used to support existing workloads. All new installations should use AVEVA Data Hub.
+
+Configuration
+-------------
+
+The configuration of the plugin is split into a number of tabs in order to reduce the size of each set of values to enter. Each tab contains a set of related items.
+
+  - **Default Configuration**: This tab contains the base set of configuration items that are most commonly changed.
+
+  - **Asset Framework**: The configuration that impacts the location with the asset framework in which the data will be placed.
+
+  - **Authentication**: The configuration required to authenticate with the OMF end point.
+
+  - **Cloud**: Configuration specific to using the cloud end points for OCS and ADH.
+
+  - **Connection**: This tab contains the configuration items that can be used to tune the connection to the OMF end point.
+
+  - **Formats & Types**: The configuration relating to how types are used and formatted with the OMF data.
+
+  - **Advanced Configuration**: Configuration of the service or task that is supporting the OMF plugin.
+
+  - **Security Configuration**: The configuration options that impact the security if the service that is running OMF.
```

Comment:
Not clear. Should this be "The configuration options that impact the security if the OMF plugin is running as a service?"

Reply:
No it should have been of the service